puppet/modules/ssh/manifests/init.pp

```diff
@@ -1,15 +1,4 @@
 class ssh {
-  file { '/root/.ssh':
-    ensure => directory,
-    owner  => 'root',
-    group  => 'root',
-  }
-
-  file { '/root/.ssh/authorized_keys':
-    ensure  => file,
```

Comment:
Should we ensure it's absent?

Reply:
Shouldn't make a difference with `PermitRootLogin no` ;)

Reply:
changed anyway